src/index.js

```diff
@@ -0,0 +1,14 @@
+//index is a naming convention that lets a user know that they are the main files.
+//The index.js component mounts the main react component onto the "root element
+import React from 'react' //imports react library(JSX) codes from 'react' folder in node-modules and
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/AndrewIndeche/Learning-Project/pullRequest?prid=9896022)
```suggestion
import React from 'react'; //imports react library(JSX) codes from 'react' folder in node-modules and
```